src/screens/TestCases/Modal/CustomFilterModal.js

```diff
@@ -110,44 +217,90 @@ const CustomFilterModal = ({
                         <select
                           className="form-select"
                           aria-label="Default select example"
+                          onChange={(e) => {
+                            setFilterType(e.target.value);
+                          }}
                         >
                           <option defaultValue>Open this select menu</option>
-                          <option value="1">Equals</option>
-                          <option value="2">Does Not Equals</option>
-                          <option value="3">Begins With</option>
+                          {filterColumn === 'id' ? (
```

Comment:
all option store into a variable, then map it

Reply:
resolved